sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleBase2ExponentialHistogramAggregator.java

```diff
@@ -48,15 +47,17 @@ public final class DoubleBase2ExponentialHistogramAggregator
   public DoubleBase2ExponentialHistogramAggregator(
       Supplier<ExemplarReservoir<DoubleExemplarData>> reservoirSupplier,
       int maxBuckets,
-      int maxScale) {
+      int maxScale,
+      MemoryMode memoryMode) {
     this.reservoirSupplier = reservoirSupplier;
     this.maxBuckets = maxBuckets;
     this.maxScale = maxScale;
+    this.memoryMode = memoryMode;
   }
 
   @Override
   public AggregatorHandle<ExponentialHistogramPointData, DoubleExemplarData> createHandle() {
-    return new Handle(reservoirSupplier.get(), maxBuckets, maxScale);
+    return new Handle(reservoirSupplier.get(), maxBuckets, maxScale, memoryMode);
   }
 
   @Override
```

Comment:
I didn't do it in the PR I'm cutting the smaller PRs out of. At first, I thought I forgot, but then I remembered I didn't do it also for the async instrument storage because the cardinality of this object is the number of instruments and not the number of attributes, so I figured this should be fine. The following PR tests the GC allocation rate, which didn't surface there; I mainly tested a high count of Attributes, not instruments.